### PR TITLE
refactor: Upgrade cloud-export-mqtt to use V3

### PR DESCRIPTION
**Pull request by marcpfuller**
**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-examples/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] I have added unit tests for the new feature or bug fix (if not, why?)
- [x] I have fully tested (add details below) this the new feature or bug fix (if not, why?)

## Testing Instructions
Start edgex with no-secty
run `./app-service -p aws -cp -o -dev`
verify service starts up
run `./app-service -p azure -cp -o -dev`
verify service starts up

### CODE REVIEW

**Review by jim-wang-intel — DISMISSED** [review]
LGTM

File: application-services/custom/cloud-export-mqtt/res/aws/configuration.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+MessageBus:
+  Disabled: false  # Set to true if not using edgex-messagebus Trigger below and Metrics are not enabled
+  Optional:
+    ClientId: "app-cloud-export-mqtt"
+
+# Choose either an HTTP trigger or edgex-messagebus trigger
+
+#Trigger:
+#Type="http"
```
Comment (lenny-intel) [inline application-services/custom/cloud-export-mqtt/res/aws/configuration.yaml:27]: Remove

File: application-services/custom/cloud-export-mqtt/res/aws/configuration.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+Trigger:
+  # Note that the MessageBus connection above is used by edgex-messagebus trigger which is the default set in common config
+  # Default value for SubscribeTopics is aslo set in common config
+  PublishTopic: "example"  # Base topic is prepened to this topic when using edgex-messagebus
```
Comment (lenny-intel) [inline application-services/custom/cloud-export-mqtt/res/aws/configuration.yaml:32]:
```suggestion
# Using default Trigger config from common config
```

File: application-services/custom/cloud-export-mqtt/res/azure/configuration.yaml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    Enabled: false
+  InsecureSecrets:
+    mqtt:
+      path: "mqtt"
+      Secrets:
+        username: "user-name"
+        password: "p@ssword"
+  Telemetry:
+    Interval: "30s"
+    PublishTopicPrefix : "edgex/telemetry" # /<service-name>/<metric-name> will be added to this Publish Topic prefix
```
Comment (lenny-intel) [inline application-services/custom/cloud-export-mqtt/res/azure/configuration.yaml:13]:
```suggestion
```

File: application-services/custom/cloud-export-mqtt/res/azure/configuration.yaml [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+    ClientId: "app-cloud-export-mqtt"
+
+# Choose either an HTTP trigger or edgex-messagebus trigger
+#Trigger:
+#Type:"http"
+
+Trigger:
+  # Note that the MessageBus connection above is used by edgex-messagebus trigger which is the default set in common config
+  # Default value for SubscribeTopics is aslo set in common config
+  PublishTopic: "example"  # Base topic is prepened to this topic when using edgex-messagebus
```
Comment (lenny-intel) [inline application-services/custom/cloud-export-mqtt/res/azure/configuration.yaml:32]:
```suggestion
# Using default Trigger config from common config
```

File: application-services/custom/cloud-export-mqtt/res/azure/configuration.yaml [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+  StartupMsg: "This is a sample cloud export to Azure"
+
+MessageBus:
+  Disabled: false  # Set to true if not using edgex-messagebus Trigger below and Metrics are not enabled
+  Optional:
+    ClientId: "app-cloud-export-mqtt"
+
+# Choose either an HTTP trigger or edgex-messagebus trigger
+#Trigger:
+#Type:"http"
```
Comment (lenny-intel) [inline application-services/custom/cloud-export-mqtt/res/azure/configuration.yaml:27]: remove

**Review by lenny-intel — CHANGES_REQUESTED** [review]

File: application-services/custom/cloud-export-mqtt/res/azure/configuration.yaml (hunk 4 above)
Comment (marcpfuller) [reply application-services/custom/cloud-export-mqtt/res/azure/configuration.yaml]: Done

File: application-services/custom/cloud-export-mqtt/res/azure/configuration.yaml (hunk 3 above)
Comment (marcpfuller) [reply application-services/custom/cloud-export-mqtt/res/azure/configuration.yaml]: Done

File: application-services/custom/cloud-export-mqtt/res/aws/configuration.yaml (hunk 2 above)
Comment (marcpfuller) [reply application-services/custom/cloud-export-mqtt/res/aws/configuration.yaml]: Done

File: application-services/custom/cloud-export-mqtt/res/aws/configuration.yaml (hunk 1 above)
Comment (marcpfuller) [reply application-services/custom/cloud-export-mqtt/res/aws/configuration.yaml]: Done

**Review by lenny-intel — APPROVED** [review]
LGTM

**Pull request merged**